src/views/ActorPageView.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="actor-card">
```

Comment:
I don't see this class being used anywhere.

Just use the `ActorDetails` component in the router, instead of having this as a separate view.

To me, ActorDetails is a view, not a component.
Let me know if you disagree.